cmd/aries-agentd/startcmd/start_test.go

```diff
@@ -333,6 +326,25 @@ func waitForServerToStart(t *testing.T, host, inboundHost string) {
 	}
 }
 
+func listenFor(host string, maxTimeToWait time.Duration) error {
+	timeout := time.After(maxTimeToWait)
+	for {
+		select {
+		case <-timeout:
+			return errors.New("timeout: server is not available")
+		default:
+			conn, err := net.Dial("tcp", host)
+			if err != nil {
+				continue
+			}
+			if err := conn.Close(); err != nil {
+				return err
+			}
+			return nil
```

Comment:
please simplify it `return conn.Close()`

Reply:
Done!